frontend/sozisel-app/src/components/PresenterSession/PresenterSession.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../styles/styles.scss";
+
+.PresenterSession {
+    height: 100%;
+    .progressIndicator {
+        width: 2*$spacing-10 !important;
```

Comment:
are you sure you have to do that nasty styling? Both important and calc?

Reply:
Yeah, it is necessary to make this progress indicator look nice.